src/services/positionService.ts

```diff
@@ -645,6 +654,16 @@ export default class PositionService {
     if (yieldFrom || yieldTo) {
       permissions = [...permissions, PERMISSIONS.TERMINATE];
     }
+    const POLYGON_WETH = getWrappedProtocolToken(destinationChainID as number);
```

Comment:
destinationChainId seems to be optional, so we should not take into account that it is actually going to exist right? Or was this a leftover from hardcoding?

Reply:
If they are on same chain, then the destination chainIDs and numbers wont be of any use, that's why kept it optional

Reply:
lets change this to not be hardcoded